File: sprint5/By languages/C#/lesson2code1.cs
using System.Collections.Generic;

public class Node {
    public int Obj { get; set; }
    public List<Node> Children { get; set; }
    
    public Node(int obj, List<Node> children = null) {
        Obj = obj;
        Children = children ?? new List<Node>();
    }
}

File: sprint5/By languages/Java/lesson2code1.java
import java.util.List;

public class Node {
    public int obj;
    public List<Node> children;
    
    public Node(int obj, List<Node> children) {
        this.obj = obj;
        this.children = children;
    }
    
    public Node(int obj) {
        this(obj, null);
    }
}
